chapters/api_design_choices.txt

```diff
@@ -97,17 +97,70 @@ A full list of ANARI extensions can be found in the <<extension_index>>
+in distriuted applications.
 
 [NOTE]
 .Note
 ====
-Interoperability with underlying
-networking APIs, such as MPI, are expected to be documented by the vendor.
+Distributed rendering define semantics and rules for ANARI usage within
+distributed _applications_. Some devices _may_ use distirbuted rendering
+resources behind the API of single-process applications, which is already
+covered by normal ANARI API usage and does not require additional specification.
+====
+
+[[api_choices_data_parallel_mpi]]
+==== MPI
+
+Extension `KHR_DATA_PARALLEL_MPI`
+
+Distributed applications using MPI can leverage ANARI implementations which also
+use MPI to enable data-parallel rendering. The overall API usage requirements
+are as follows:
+
+- Applications collectively create the device, frame, and world in lockstep.
+- Each rank creates/modifies the local contents of the
+  <<object_types_world,World>> it has using the ANARI API normally.
+- All ranks collectively render by calling `anariRenderFrame` in lockstep.
+
+Given an MPI distributed application using ANARI, all ANARI concepts apply along
+with the following additional rules:
+
+- Applications must create <<object_types_frame,Frame>> and
+  <<object_types_world,World>> objects in the same order. Construction order
+  they are collectively identified among all ranks.
+- Participating <<object_types_frame,Frame>>,
+  <<object_types_renderer,Renderer>>, and <<object_types_camera,Camera>>
+  parameters must match on all ranks. Using different parameters for these
+  objects on different ranks will result in undefined behavior.
+- Mapping frame channels is only well-defined on a designated main rank.
+  While it is not an error to map any channel on any other rank, its contents
+  (size, pixel type, and returned data pointer) are undefined.
+- Properties queried with `anariGetProperty` are globally synchronizing
+  (collective) operations for <<object_types_frame,Frame>> and
+  <<object_types_world,World>> objects when queried with the `ANARI_WAIT` flag.
+- Calling `anariRelease` on the final application held reference of the
+  <<anari_concepts_devices,Device>> is a globally synchronized (collective)
+  operation.
+
+This extension introduces the following additional parameters:
+
+.<<api_concepts_parameters, Parameters>> added to MPI distributed <<object_types_frame,Frame>> objects.
+[cols="<2,<,^,<20",options="header,unbreakable"]
+|===================================================================================================
+| Name     | Type         | Default | Description
+| mainRank |`INT32`       | 0       | Designated application rank which will map frame channels.
+|===================================================================================================
+
+[NOTE]
+.Note
+====
+A more complete description of the problem, ANARI-based solution, and example
+implementation results can be found in the freely available paper
+"Standardized Data-Parallel Rendering using ANARI":
```

Comment:
Let's use the same format as in Sec "Normative Refs", i.e.,
Author. Date. *Title*, url.
(I just see we are not 100% consistent there either...)

Reply:
I _think_ this is addressed in latest version, let me know if the format is correct.